.travis.yml

```diff
@@ -7,15 +8,16 @@ dist: trusty
 python:
   - "2.7"
 
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y hostapd dnsmasq libdbus-1-dev libdbus-glib-1-dev
```

Comment:
Are we okay with removing libdbus-1-dev and install python-dbus only?

Reply:
I found that we have made a similar discussion here: https://github.com/wifiphisher/wifiphisher/pull/610#issuecomment-307324134

It looks unnecessary to me but let me know if you or @blackHatMonkey think otherwise.

Reply:
Ya ok what ever that means

Reply:
dbus-python requires lib-dbus as dependency. I remembered @blackHatMonkey has to install this to import dbus?